### ISSUE TITLE
Crash loading example__scripting.bsk on OSX

### ISSUE THREAD

**Issue by timnewsham** [BespokeSynth/BespokeSynth#129]
When I launch bespoke and load the "example__scripting.bsk" bespoke crashes immediately without any output to the console other than "Killed: 9".
The full console output is:

`objc[53247]: Class FIFinderSyncExtensionHost is implemented in both /System/Library/PrivateFrameworks/FinderKit.framework/Versions/A/FinderKit (0x7fffad0143d8) and /System/Library/PrivateFrameworks/FileProvider.framework/OverrideBundles/FinderSyncCollaborationFileProviderOverride.bundle/Contents/MacOS/FinderSyncCollaborationFileProviderOverride (0x190e94f50). One of the two will be used. Which one is undefined.
Killed: 9`

but the first line is printed before I click the "load" button.

`Darwin TimBookAir.local 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64`
OSX 10.14.6

**Comment by JulianHeuser** [comment]
I'm also getting a crash when loading the example on Windows 10, all other examples work fine. There's no console output afaik (is there a way to enable that?) but I have a crash log:

```
stack frame:
PyEval_GetGlobals + 0x11
CallWindowProcW + 0x3f8
DispatchMessageW + 0x259
BaseThreadInitThunk + 0x14
RtlUserThreadStart + 0x21

backtrace:
3: UnhandledExceptionFilter + 0x1e7
4: memset + 0x13b0
5: _C_specific_handler + 0x96
6: _chkstk + 0x11f
7: RtlRaiseException + 0x434
8: KiUserExceptionDispatcher + 0x2e

OS: Windows 10
CPU vendor: AuthenticAMD
CPU model: AMD Ryzen 3 2200G with Radeon Vega Graphics
CPU speed: 3492 MHz
num cores: 4
num CPUs: 4
RAM: 16329 MB
computer: HAL
language: en
region: US
display language: en-US
description: Windows (Desktop)
manufacturer: 
build: bespoke 1.0.0 (Sep 13 2021 20:28:53)
```

**Comment by awwbees** [comment]
do you get a crash when spawning in a "script" module, or just when loading that savestate?

if so, does it crash immediately upon spawning the "script" module, or only after clicking through the warning?

if immediately, do you see a file at `Documents\BespokeSynth\internal\python_installed`?

**Comment by KottV** [comment]
i found that `example__scripting.bsk` is crashing on load with python 3.9 on Linux, with 3.8 it loads and runs fine

**Comment by timnewsham** [comment]
The crash happens as soon as I load the script.
`$ ls ~//Documents/BespokeSynth/internal`
`./	../`

empty dir

**Comment by boonier** [comment]
is version 3.8.6 ok? If so, I'm still getting immediate crash on macOS 10.14.6

no file in Documents\BespokeSynth\internal\python_installed

cheers

**Comment by awwbees** [comment]
fixed in 5293c11c6b46ea35b500d2be780e7d049a20a350
